packages/sdk-v2/src/sdk/ipfs/utils/async.ts

```diff
@@ -0,0 +1,34 @@
+import { first, isFunction, noop } from "lodash";
```

Comment:
don't we have some common utils ? somewhere in the core ? 
lets move those functions there

Reply:
ok moved to root src